src/unnamed_rcv_thing/cvr_loader.py

```diff
@@ -0,0 +1,81 @@
+import pandas as pd
+import pathlib
+import os
+from pydantic import BaseModel
+from typing import Callable
+from unnamed_rcv_thing.profile import PreferenceProfile
```

Comment:
when importing files from within the package, you can do `from .profile import PreferenceProfile` instead of repeating the package name (makes future refactors easier)